spyderlib/widgets/dicteditor.py

```diff
@@ -1075,7 +1076,23 @@ def copy(self):
         for idx in self.selectedIndexes():
             if not idx.isValid():
                 continue
-            clipl.append(to_text_string(self.delegate.get_value(idx)))
+            obj = self.delegate.get_value(idx)
+            # check if it's a numpy array, and if so make sure to copy the whole
+            # thing in a tab separated format
+            try:
+                import numpy as np
+            except ImportError:
+                pass # skip this part

```

Comment:
I think this `try/except` is not needed. If Numpy arrays are shown in the Variable Explorer, that means `numpy` is already present, so we don't need to verify for its presence.